doc/geany.txt

```diff
@@ -1785,6 +1786,25 @@ This works basically the same as on other platforms::
     "c:\program files\geany\bin\geany" -g c:\mytags.php.tags c:\code\somefile.php
 
 
+Generating tags files using ctags
+*********************************
+Geany supports loading tag files generated using the ``ctags`` command-line
+tool from the universal-ctags project (https://github.com/universal-ctags/ctags).
+Even though Geany should work with any ctags file, it is recommended to use
+certain fields to give Geany some additional information. The recommended fields
+are ``EfiklsZSt``, so, for instance, to generate tags for all sources in the geany
+directory, one can use::
+
+    ctags --fields=EfiklsZSt -o geany.c.tags -R geany
+
+Additional options may be given to the ``ctags`` tool, for instance, to restrict
+the generated tags file to some languages only, use certain tag kinds, etc.
+
+Note that when the ``l`` field (specifying the programming language for every tag)
+is enabled, the language of the tag is set based on this field instead of the
```

Comment:
question, is it the tag or the symbol in Geany that gets the language specified?

Reply:
Oh, my response from yesterday disappeared somewhere. So once again - you mean whether to use the word "tag" or "symbol" here, right? I checked geany.txt and we only use "tags" in "tags file" but never talk about tags themselves so probably "symbol" is the correct one here. I've modified it to:

```
Note that when the ``l`` field (specifying the programming language) is enabled,
the language of all symbols is set based on the value of this field instead of
the language specified in the extension of the tags file.
```

Does it sound OK?

Reply:
Yep fine.

Reply:
Thanks, repushed with all the suggested changes.